src/bindings/python/tests/test_graph/test_ops.py

```diff
@@ -117,9 +117,13 @@ def test_broadcast():
 @pytest.mark.parametrize("node", [
     Constant(Type.f32, Shape([3, 3]), list(range(9))),
     ov.constant(np.arange(9).reshape(3, 3), Type.f32),
-    ov.constant(np.arange(9).reshape(3, 3), np.float32)
+    ov.constant(np.arange(9).reshape(3, 3), np.float32),
+    None
```

Comment:
I suggest to rewrite a little bit this test using [this feature](https://docs.pytest.org/en/7.1.x/example/parametrize.html#parametrizing-conditional-raising).

```suggestion
    ov.constant(None)
```